packages/edit-site/src/components/page-pages/index.js

```diff
@@ -96,43 +96,64 @@ export default function PagePages() {
 				id: 'featured-image',
 				header: __( 'Featured Image' ),
 				getValue: ( { item } ) => item.featured_media,
-				render: ( { item, view: currentView } ) =>
-					!! item.featured_media ? (
-						<Media
-							className="edit-site-page-pages__featured-image"
-							id={ item.featured_media }
-							size={
-								currentView.type === 'list'
-									? [ 'thumbnail', 'medium', 'large', 'full' ]
-									: [ 'large', 'full', 'medium', 'thumbnail' ]
-							}
-						/>
-					) : null,
+				...coreFieldTypes.image,
```

Comment:
Could you clarify why we do this if `featured-image` defines its own `render`?

Reply:
For now it doesn't do more, but the goal is to have more default things like `edit`. Maybe it's makes more sense to try to see how the `edit` would be here and evaluate again. I'll convert to draft.

Reply:
👍